### ISSUE TITLE
Sending a tweet from your program

### ISSUE THREAD

**Issue by darthcoder** [mikaturunen/electron-twitter#2]
Hi, I'm a newbie, can you suggest how to extend your program to send a tweet? I think that it could be done with the twitter package (that is a dependency of your project). I think there would need to be a form to take input, but I cannot figure out how (and where) to save the form data.

**Comment by mikaturunen** [comment]
1. Create html input + button and hook on-click event to the button: create two new html elements, a bit of javascript into the html file to hook to the on-click event. Random on click example: http://www.w3schools.com/jsref/tryit.asp?filename=tryjsref_onclick_html
2. Send the input `.val()` to the node/io process via IPC: https://github.com/electron/electron/blob/master/docs/api/ipc-renderer.md (documentation here on how to use it).
3. Capture the IPC event on node side and send the input into the twitter api https://github.com/electron/electron/blob/master/docs/api/ipc-main.md (documentation on how to capture the ipc on node side).
4. Send tweet, example here https://www.npmjs.com/package/twitter (find "How about an example that passes parameters? Let's tweet something:"). 

```
client.post('statuses/update', {status: 'I Love Twitter'},  function(error, tweet, response) {
  if(error) throw error;
  console.log(tweet);  // Tweet body. 
  console.log(response);  // Raw response object. 
});
```

So essentially you don't have to "save" the tweet per se, just send it away. If you want to see them, you just use the twitter package to list your own tweets (refresh them to see the new tweet). 

Hope this helps a bit. I know it can be a bit daunting but I'll help along as much as I can :)

**Comment by darthcoder** [comment]
thanks so much for your reply. i have some background in python and c but i am very green at web tech. electron seems interesting and i am trying to take one bite sized chunk out of it at a time. 

also the version of electron that you have as a dependency is 0.25.x while the newest is at 1.4 or something. should i update that string to the latest version in my fork?

**Comment by mikaturunen** [comment]
Yes, definitely. It won't hurt to update. You know instantly if it doesn't work 👍

**Comment by darthcoder** [comment]
ok so I just changed package.json dependency and dev-dependency to 1.4.0 and tried to run 
`npm install && npm start` and I got a metric tonne of errors about a possible memory leak and further it said:

```
  App threw an error during load
 Error: Cannot find module 'app'
```

then I looked at your `main.js` and compared it with the `main.js` of the [electron quick start](https://github.com/electron/electron-quick-start) project and I noticed that their starting lines are a bit different: 

```
   const electron = require('electron')
   // Module to control application life. 
   const app = electron.app
```

should i try to change the lines? i am not sure about how everything will fit after that.
